### DISCUSSION TITLE
8월 3일 : 회의록

### DISCUSSION THREAD

**Discussion by numerical43** [cca-ffodregamdi/running-hi-back#2] [Ideas]
### 회의 내용 정리
- 피그마는 거의 다 공통 컴포넌트로 (페이지 버튼, 템플릿, 헤더, 푸터, 관리자 페이지의 좌측 메뉴 등)
- 메인화면 좌측에 영어 로고
- 메인화면의 러너들의 추천 → 사진과 장소 이름이 간략하게 보이게
- 오른쪽에 햄버거 버튼 & 클릭하면 우측에서 슬라이드로 메뉴 나타나게
    - 마이 페이지, 마이 피드, 로그아웃,  러닝하이 추천, 러너들의 추천
- 러너들의 추천 :  사진 1개만, 경로 파일 다운로드, 경로를 map으로 제공 (**지도에서 경로 전체가 다 보일 수 하는게 중요)**
    - 거리, 평균 고도, 난이도, 예상 시간
- 러닝하이 추천 : 순위(계속 새 파일 올릴 수 있음, 새로 올리면 기존 데이터 삭제), 경로 파일 다운로드

<br>

### 자료 조사
서울 : 종완 (5개)
경기 - 강원 - 충청북도 : 수의
대구 - 대전 - 전라북도 : 만제
경상남도 - 광주 - 전라남도 : 효진
제주 - 인천 - 충청남도 : 현지
부산 - 세종 - 경상북도 : 정인

**Comment by Subak-Uncle** [comment]
잠깐만요! 지역이 

서울 : 종완 (5개)
경기 - 강원 - 충청북도 : 수의 
대구 - 대전 - 전라북도 : 만제
경상남도  - 광주 - 전라남도 : 효진
제주 - 인천 - 충청남도  : 현지
 부산 - 세종 - 경상북도 : 정인
 
로 수정해주세요!

각 지역 당 1개 씩 주세요!

**Reply by numerical43** [reply]
네 알겠습니다!
